guarddog/analyzer/sourcecode/download-executable.yml

```diff
@@ -62,18 +62,42 @@ rules:
                             pattern-regex: (?i)^['"].*?\.exe['"]$
 
     pattern-sources:
-      - pattern: (...).send(...)
-      - pattern: send(...)
-      - pattern: (...).request(...)
-      - pattern: request(...)
-      - pattern: (...).urlopen(...)
-      - pattern: urlopen(...)
-      - pattern: (...).getresponse(...)
-      - pattern: getresponse(...)
-      - pattern: requests.$FUNC(...)
-      - pattern: (...).urlretrieve(...)
-      - pattern: urlretrieve(...)
-      - pattern: requests.get(...)
+      - pattern: |
+          (...).send(...)
+          ...
```

Comment:
is this needed?

Reply:
actually i did not investigate why this pattern was present, just appended the ellipsis to all of them to match the new case.
git blame shows this was the in the first commit of this file.
No test is using it, should I remove it?

Reply:
unless we have an evidence it's needed, I would be in favor of removing it

Reply:
fixed in 57348bffa7101cb2ff4e88f6dd77fdb77283b32c